pages.ar/windows/curl.md

```diff
@@ -1,7 +1,7 @@
 # curl
 
 > هذا الأمر هو اسم مستعار لـ `curl -p common`.
-> لمزيد من التفاصيل: <https://curl.se>.
+> لمزيد من التفاصيل: <https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/invoke-webrequest>.
```

Comment:
Double checking here, if this is a typo.

Reply:
It is intentional as the curl command is an alias for Invoke we request in Windows.

Reply:
Ahh! TiL
Thanks